remo/base/utils.py

```diff
@@ -142,6 +142,15 @@ def get_date(days=0, weeks=0):
             datetime.timedelta(days=days, weeks=weeks))
 
 
+def get_date_from(date=timezone.now().date(), days=0, weeks=0):
```

Comment:
I would propose to drop this helper completely and do the timedelta calculations on the fly.